### ISSUE TITLE
Build instructions broken

### ISSUE THREAD

**Issue by ogra1** [snapcore/pi-gadget#63]
Trying to build a gadget following the instructions in README.md results in:

```
$ sudo snapcraft snap --target-arch=arm64 --destructive-mode
Running with 'sudo' may cause permission errors and is discouraged. Use 'sudo' when cleaning.
Setting target machine to 'arm64'
--target-arch has been deprecated and is no longer supported on core20.
$
```

The gadget should be reverted to a proper snapcraft.yaml again (with proper parts to do all porocessing as it used to be) instead of moving back a decade in time to shell out to a Makefile. Then using the required ```architectures:``` fields for cross building will work again as well, there is nothing in the Makefile that can not be done cleaner from snapcraft parts.

**Comment by bugraaydogar** [comment]
Currently, the best way to build is;
`snapcraft --use-lxd`

**Comment by waveform80** [comment]
Fixed in #75 for core branches, and d2f04d52f92d528799877ae2eaa116321f11f2f3 for the classic branch
